apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import styles from './chatMessage.module.scss';
+import classNames from 'classnames';
+
+interface ChatMessageProps {
+  id: string;
+  name: string;
+  role: string;
```

Comment:
If we know that `role` can only be `"assistant"` or `"student"`, we can just make this type `role: 'student' | 'assistant'` to guarantee it's one of the two.

Reply:
Cool - will do!